Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
 import './Header.css';
+import ImageThumbnail from './ImageThumbnail';
+import HeaderContent from './HeaderContent';
+
+function HeaderContainer (props) {
+  return (
+    <div className="container">
+      <ImageThumbnail />
+      <HeaderContent />
```

Comment:
I think the idea for this project is passing props. So you would write line 10 like ` <HeaderContent  headerParagraph="Let's learn React by building simple Interfaces with components. Don't try to overthink it, just keep it simple and have fun. Once you feel comfortable using components you are well on your way to mastering React!" />` then use it in the HeaderContent file.